src/GlobalStyles.js

```diff
@@ -1,18 +1,22 @@
 import {createGlobalStyle} from 'styled-components';
-
 // Trick prettier into formatting "createGlobalStyle"
 const styled = {createGlobalStyle};
 
 export default styled.createGlobalStyle`
   * {
     box-sizing: border-box;
-    margin: 0;
     padding: 0;
+    max-width: 800px;
+    margin: 0 auto;
```

Comment:
Do you really want **every element** to have `margin: 0 auto` and `max-width: 800px`?
It would be better to use the `*` selector for a css reset (e.g. `padding:0; margin: 0;`) and to do the specific styling where needed.

Reply:
yes i did the whole thing for now so that it looks like anything at all. without that everything is left side bound. from thursday i'll implement the styling